run/GCHP/HISTORY.rc.templates/HISTORY.rc.fullchem

```diff
@@ -1760,7 +1760,7 @@ COLLECTIONS: @#'DefaultCollection',
 #
 # Archives chemical reaction rates from the KPP solver.
 # It is best to list individual reactions to avoid using too much memory.
-# Reactions should be listed as "RxnRate_EQnnn", where nnn is the reaction
+# Reactions should be listed as "RxnRate_EQnnnn", where nnnn is the reaction
```

Comment:
We would also have to make similar updates in:

- `run/GCHP/HISTORY.rc.templates/HISTORY.rc.carbon`

as this simulation also uses a KPP_based mechanism.

Reply:
This template files currently don't contain `RxnRate` or `RxnConst` collections.